src/drivers/localstorage.js

```diff
@@ -39,6 +39,10 @@
 
         dbInfo.keyPrefix = dbInfo.name + '/';
 
+        if (dbInfo.storeName !== self._defaultConfig.storeName) {
+            dbInfo.keyPrefix += dbInfo.storeName + '/';
+        }
```

Comment:
I remember someone had reported this somewhere....
Anyway, did you also add a test case for this?